lib/chat/types.js

```diff
@@ -55,7 +55,7 @@ export type ChatOptionsSearch = {|
 
 /**
  */
-export type ChatListOptionals = {|
```

Comment:
I think `Optionals` should be `Options`?

Reply:
SGTM